docs/Dropbox/README.md

```diff
@@ -0,0 +1,109 @@
+# Federated Login with Dropbox
+
+Start by going to the [Dropbox App console page](https://dropbox.com/developers/apps/). Click the 'Create App' button. Choose your API, access type, and name, then click 'Create App':
+
+![Create the app](create-application.png)
+
+Fill in the rest of the app information, particularly the Redirect URIs:
+
+![Redirect URI](callback-url.png)
+
+Now that we have an OAuth application registered with Dropbox, we can add Imperial to our project (We will not be going over how to create the project, as I will assume that you have already done that).
+
+Add the following line of code to your `dependencies` array in your package manifest file:
+
+```swift
+.package(url: "https://github.com/vapor-community/Imperial.git", from: "0.5.3")
+```
+
+**Note:** There might be a later version of the package available, in which case you will want to use that version.
+
+You will also need to add the package as a dependency for the targets you will be using it in:
+
+```swift
+.target(name: "App", dependencies: ["Vapor", "Imperial"],
+               exclude: [
+                   "Config",
+                   "Database",
+                   "Public",
+                   "Resources"
+               ]),
+```
+
+Then run `vapor update` or `swift package update`. Make sure you regenerate your Xcode project afterwards if you are using Xcode.
```

Comment:
I'm of the opinion that all of this should move to Xcode 11's built in SwiftPM support and just edit and save the manifest.